server/cliserver_test.go

```diff
@@ -86,3 +92,63 @@ func TestGetDelegatorInfo(t *testing.T) {
 	req.Nil(err)
 	assert.Equal("{}", string(body))
 }
+
+func TestRegisteredOrchestrators(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+	dbh, dbraw, err := common.TempDB(t)
+	require.Nil(err)
+	defer dbh.Close()
+	defer dbraw.Close()
+
+	addr := pm.RandAddress()
+
+	orch := &common.DBOrch{
+		ServiceURI:        "foo",
+		EthereumAddr:      addr.Hex(),
+		PricePerPixel:     5000,
+		ActivationRound:   100,
+		DeactivationRound: 1000,
+		Stake:             100000,
+	}
+
+	err = dbh.UpdateOrch(orch)
+	require.Nil(err)
+
+	eth := &eth.StubClient{
+		Orchestrators: []*lpTypes.Transcoder{
+			{Address: addr,
+				Active:         true,
+				DelegatedStake: big.NewInt(100000),
+				RewardCut:      big.NewInt(5),
+				FeeShare:       big.NewInt(10),
+				ServiceURI:     orch.ServiceURI},
+		},
+	}
+
+	n, _ := core.NewLivepeerNode(eth, "./tmp", dbh)
+
+	s := NewLivepeerServer("127.0.0.1:1938", n)
+	mux := s.cliWebServerHandlers("addr")
+	srv := httptest.NewServer(mux)
+	defer srv.Close()
+
+	res, err := http.Get(fmt.Sprintf("%s/registeredOrchestrators", srv.URL))
```

Comment:
Let's add a test for when `SelectOrchs()` returns an empty slice resulting in the orchestrator's default `PricePerPixel` being set to `big.NewRat(0, 1)`?